.changeset/sharp-suits-think.md

```diff
@@ -0,0 +1,8 @@
+---
+'@directus/specs': minor
+'tests-blackbox': minor
+'docs': minor
+'@directus/api': minor
+---
+
+Support multipart/form-data to apply schema diffs
```

Comment:
We opted to compose changesets in past tense:
```suggestion
Added support for the `multipart/form-data` content type in the `/schema/apply` endpoint
```

Reply:
oh I see!